src/tests/ftest/scrubber/snapshot.py

```diff
@@ -0,0 +1,51 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from scrubber_test_base import TestWithScrubber
+
+
+class TestScrubberEvictWithSnapshot(TestWithScrubber):
+    # pylint: disable=too-many-nested-blocks
```

Comment:
```suggestion
```

Reply:
I will check it... On one my PR runs, it will expecting these #pylint... Not sure whether it was this PR.

Reply:
This pylint is usually when you have too many nested blocks like:
```
if conditionA:
    if conditionB:
        if conditionC:
            if conditionD:
                if conditionE:
```

Reply:
Agree. I am not sure whether the problem is how TestScrubberEvictWithAggregation class inherits TestWithScrubber (scrubber_test_base.py) which in turn inherits IorTestBase... I don't see a deep if condition sequence...

Anyway, I will once again take a look at the issue.

Reply:
Done. Removed the #pylint.